radicle-cli/src/commands/inspect.rs

```diff
@@ -0,0 +1,222 @@
+    the current project is inspected.
+
+Options
+
+    --id-only   Return the ID in simplified form
+    --payload   Inspect the object's payload
+    --refs      Inspect the object's refs on the local device (requires `tree`)
+    --history   Show object's history
+    --help      Print help
+"#,
+};
+
+#[derive(Default, Debug, Eq, PartialEq)]
+pub struct Options {
+    pub path: Option<PathBuf>,
+    pub id: Option<Id>,
+    pub refs: bool,
+    pub payload: bool,
+    pub history: bool,
+    pub id_only: bool,
+}
+
+impl Args for Options {
+    fn from_args(args: Vec<OsString>) -> anyhow::Result<(Self, Vec<OsString>)> {
+        use lexopt::prelude::*;
+
+        let mut parser = lexopt::Parser::from_args(args);
+        let mut path: Option<PathBuf> = None;
+        let mut id: Option<Id> = None;
+        let mut refs = false;
+        let mut payload = false;
+        let mut history = false;
+        let mut id_only = false;
+
+        while let Some(arg) = parser.next()? {
+            match arg {
+                Long("help") => {
+                    return Err(Error::Help.into());
+                }
+                Long("refs") => {
+                    refs = true;
+                }
+                Long("payload") => {
+                    payload = true;
+                }
+                Long("history") => {
+                    history = true;
+                }
+                Long("id-only") => {
+                    id_only = true;
+                }
+                Value(val) if path.is_none() && id.is_none() => {
+                    let val = val.to_string_lossy();
+
+                    if let Ok(val) = Id::from_str(&val) {
+                        id = Some(val);
+                    } else if val.starts_with("rad:git:") {
+                        return Err(anyhow!("invalid ID '{}'", val));
+                    } else if let Ok(val) = PathBuf::from_str(&val) {
+                        path = Some(val);
+                    } else {
+                        return Err(anyhow!("invalid path or ID '{}'", val));
+                    }
+                }
+                _ => return Err(anyhow::anyhow!(arg.unexpected())),
+            }
+        }
+
+        Ok((
+            Options {
+                id,
+                path,
+                payload,
+                history,
+                refs,
+                id_only,
+            },
+            vec![],
+        ))
+    }
+}
+
+pub fn run(options: Options, ctx: impl term::Context) -> anyhow::Result<()> {
+    let profile = ctx.profile()?;
+    let storage = &profile.storage;
+    let signer = term::signer(&profile)?;
+
+    let id = options
+        .id
+        .or_else(|| {
+            radicle::rad::repo(options.path.unwrap_or_else(|| Path::new(".").to_path_buf()))
+                .ok()
+                .map(|(_, id)| id)
+        })
+        .context("Couldn't get ID / Path from command line and cwd is not a reposiitory either")?;
```

Comment:
```suggestion
        .context("Supplied argument is neither a valid path, nor a valid id")?;
```

Reply:
We shouldn't be trying to inspect CWD if an argument is supplied

Reply:
Well we aren't, in order we look for:
1- `id` supplied in `rad inspect <id>`
2- then, `path` supplied in `rad inspect <path>`
3- and then if path is `None`, we use `.`

Reply:
Ok then just the msg change will do it.

Reply:
But the msg doesn't say anything about cwd, btw the msg is from original [rad-edit](https://github.com/radicle-dev/radicle-cli/blob/master/edit/src/lib.rs#L75)

Reply:
Why would it say anything about the CWD if you passed an argument?